tests/integrations/raftstore/test_merge.rs

```diff
@@ -781,7 +786,8 @@ fn test_merge_with_slow_promote() {
     must_get_equal(&cluster.get_engine(3), b"k1", b"v1");
     must_get_equal(&cluster.get_engine(3), b"k3", b"v3");
 
-    let delay_filter = Box::new(DelayFilter::new(Duration::from_millis(20)));
+    let delay_filter =
+        Box::new(RegionPacketFilter::new(right.get_id(), 3).direction(Direction::Recv));
```

Comment:
Why does it stable the test?

Reply:
`DelayFilter` will make some threads sleep, which is not expected here. Just drop those messages is enough.